### PR TITLE
fix epel repo

### PR DESCRIPTION
**Pull request by jcantrill**

### CODE REVIEW

**Comment by jcantrill** [comment]
[test]

File: curator/install.sh [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 set -ex
 
-rpm -q epel-release || yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+yum makecache fast
```
Comment (smarterclayton) [inline curator/install.sh]: I don't know that you should add this - we generally try not to in our Dockerfiles

**Comment by jcantrill** [comment]
[test]

**Comment by jcantrill** [comment]
[merge]

**Comment by jcantrill** [comment]
[merge]

**Pull request merged**